src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -148,10 +151,70 @@ def init_db():
     """
     # SyncD database connector. THIS MUST BE INITIALIZED ON A PER-THREAD BASIS.
     # Redis PubSub objects (such as those within swsssdk) are NOT thread-safe.
-    db_conn = SonicV2Connector(**redis_kwargs)
+    db_conn = SonicV2Connector(**redis_kwargs) 
 
     return db_conn
 
+def init_namespace_dbs():
+    db_conn= []
+    # Global db connector
+    SonicDBConfig.load_sonic_global_db_config()
+    for namespace in SonicDBConfig.get_ns_list():
+        db = SonicV2Connector(use_unix_socket_path=True, namespace=namespace)
+        db_conn.append(db)
+
+    return db_conn
+
+def connect_all_namespace_dbs(all_ns_db, db_id):
+    for db_conn in all_ns_db:
+        db_conn.connect(db_id)
+
+def get_keys_from_all_namespace_dbs(all_ns_db, db_id, pattern='*'):
+    """
+    db keys function execute on global and all namespace DBs.
+    """
+    result_keys=[]
+    for db_conn in all_ns_db:
+        db_conn.connect(db_id)
+        keys = db_conn.keys(db_id, pattern)
+        if keys is not None:
+            result_keys.extend(keys)
+    return result_keys
+
+
+def get_all_from_namespace_dbs(all_ns_db, db_name, _hash, *args, **kwargs):
+    """
+    db get_all function executed on global and all namespace DBs.
+    """
+    for db_conn in all_ns_db:
+        db_conn.connect(db_name)
+        if(db_conn.exists(db_name, _hash)):
+            return db_conn.get_all(db_name, _hash, *args, **kwargs)
+    return {}
+
+def get_bridge_port_map_from_namespace_dbs(all_ns_db, db_name):
+    """
+    get_bridge_port_map from all namespace DBs
+    """
+    if_br_oid_map = {}
+    if len(all_ns_db) == 1:
```

Comment:
To prevent the overhead to check multiple namespace each time, how about add a new class for `all_ns_db`, and normalize the db accessing for both use cases?

Reply:
Added a new class.